src/pages/UserPage/User.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import JobDetails from '../Home/JobDetails';
+import JobFeed from '../Home/JobFeed';
+import FormData from '../../components/DataModels/FormData';
+import axios from 'axios';
+import { REACT_BACKEND_URL } from '../../config';
+import { useCookies } from 'react-cookie';
+
+const User=()=>{
+  const [jobs,setJobs]=React.useState([]);
+  const [currentJob,setCurrentJob]= React.useState(null);
+  const [view,setView]= React.useState('hide');
+  const [ authCookie, setAuthCookie, removeAuthCookie] = useCookies([]);
+  const jobClick=(job:FormData,currentView:string)=>{
+    setView(currentView);
+    setCurrentJob(job);
+  };
+  React.useEffect(()=>{
+    const fetchData=async ()=>{
+      await axios.get(`${REACT_BACKEND_URL}/v1/job/${authCookie.email}`).then(res=>{setJobs(res.data);});
```

Comment:
this api call should come from service file.  and in the api we should not send email id. please talk to nagaraju or pranav.